janusgraph-test/src/main/java/org/janusgraph/JanusGraphDatabaseProvider.java

```diff
@@ -0,0 +1,67 @@
+package org.janusgraph;
```

Comment:
Please add the standard copyright header at the top of this and other new files.

Reply:
I see this was done.